pkg/hubble/server/local_observer.go

```diff
@@ -143,6 +151,7 @@ nextEvent:
 			}
 		}
 
+		s.numObservedFlows++
```

Comment:
Since you increment this counter here, the reported observed flows might be different than what was actually sent to the ring buffer. Is this what we want? Do we want a "processed flows" variable to differentiate the two?

Reply:
That's a good point and something that actually already bit me, since this also includes events currently ignored by Hubble (e.g. agent notifications). I will move it.